libraries/manifest/page_metrics.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import print_function, unicode_literals
```

Comment:
Not going to make this a required change, but just discussion as maybe you have good reason for it. 

My thinking its that since the page view count is part of door43 it should be in the door43_tools library, not a new manifest library, since it is a tool for making door43 do what it does. 

In the tX pipeline we have four separate entities that deal with their own part: DCS(Gitea) --> Client --> Manager/Converters --> Client -->  Door43. Page views is part of that Door43 piece. Thoughts? Just don't see "manifest" as self-contained entity in the pipeline.